introduction.md

```diff
@@ -3,3 +3,11 @@ Hello I am royce, it's nice to meet you!
 I have not gone though all of the courses so I can only choose my favorite based on
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?